testing/tests/DevExpress.ui.widgets.scheduler/timeline.tests.js

```diff
@@ -1169,6 +1169,22 @@ QUnit.test("TimelineWorkWeek has right count of cells with view option intervalC
     assert.equal(cells.length, this.instance._getCellCountInDay() * 5 * 4, "view has right cell count");
 });
 
+QUnit.test("'getCoordinatesByDate' should return right coordinates with view option intervalCount", function(assert) {
+    this.instance.option({
+        intervalCount: 2,
+        currentDate: new Date(2017, 5, 25),
+        startDayHour: 8,
+        endDayHour: 20
+    });
+
+    var $element = this.instance.element();
+
+    var coords = this.instance.getCoordinatesByDate(new Date(2017, 6, 6, 12, 0), 0, false);
+
+    assert.equal(coords.top, $element.find(".dx-scheduler-date-table tbody td").eq(200).position().top, "Cell coordinates are right");
```

Comment:
Extract a variable for this `$element.find(".dx-scheduler-date-table tbody td").eq(200).position()`